### DISCUSSION TITLE
Redisson polluting logs

### DISCUSSION THREAD

**Discussion by gsinhazscaler** [redisson/redisson#5669] [General]
I have Redisson in Info Mode. But I still see this in the log. Any clue how to get rid of it
<img width="1127" alt="Screenshot 2024-03-05 at 12 44 21 PM" src="https://github.com/redisson/redisson/assets/90656654/8fb370e2-038c-4d82-9139-ef60f7abe1d7">

**Comment by gsinhazscaler** [comment]
06 Mar 2024 00:38:57.856 UTC [redisson-netty-1-32] DEBUG i.n.h.s.SslHandler -  [id: 0xac19bebd, L:/10.66.20.84:31028 - HANDSHAKEN: protocol:TLSv1.2 cipher suite:TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256

**Reply by mrniko** [reply]
it looks like logging level for io.netty package is `debug`
